Dockerfile

```diff
@@ -356,7 +356,7 @@ ENV BUILDKIT_SETUP_CGROUPV2_ROOT=1
 ENTRYPOINT ["buildkitd"]
 
 FROM buildkit-linux AS buildkit-linux-debug
-COPY --link --from=dlv /usr/bin/dlv /usr/bin/dlv
+COPY --link --from=dlv /out/dlv /usr/bin/dlv
```

Comment:
@crazy-max Looks like this broke in https://github.com/moby/buildkit/pull/5497/files . We should have a CI check to make sure it doesn't go unnoticed.